meteor/client/ui/Settings/ShowStyle/VariantSettings.tsx

```diff
@@ -35,7 +42,101 @@ export const ShowStyleVariantsSettings = withTranslation()(
 
 			this.state = {
 				editedMappings: [],
+				uploadFileKey: Date.now(),
+			}
+		}
+		importVariants(e: React.ChangeEvent<HTMLInputElement>) {
```

Comment:
This method is 50ish lines long whicn should be an indication that it could be refactored into smaller methods.
A quick one would be refactoring the body of your foreach into its own method.

Reply:
Changed